content/articles/cepos/cepos_001.rst

```diff
@@ -0,0 +1,13 @@
+Eliens' First Quest of 2020: Algolympics
+#########################################
+
+:date: 2020-2-3
+:category: News
+:author: Clinton E. Poserio
+
+Eliens, the moniker of the UPLB Competitive Programming team, is back in action as they qualify for the Final Round of Algolympics 2020, a programming contest organized by `ACM-UP Diliman Student Chapter, Inc.<https://www.facebook.com/upacm>`_. Four teams from Eliens, with three individuals each, were formed and registered to the Online Elimination Round of Algolympics 2020 last January 17-27, 2020. The ten-day long contest was held on the Codeforces_ platform, and was participated by 83 teams[1]_ from 20 different universities in the Philippines. However, only the top 20  teams  from all the participating teams and only the top three teams from each university are qualified to join the final round. With the skills of these students, all three slots for UPLB were filled; three teams of Eliens are qualified to the Final Round of Algolympics 2020. 
```

Comment:
With the skills of the participating students, all three slots for UPLB were filled; three teams from Eliens qualified to the Final Round of Algolympics 2020.